mix.exs

```diff
@@ -55,11 +63,14 @@ defmodule TaskBunny.Mixfile do
 
   defp deps do
     [
-      {:poison, "~> 2.0"},
       {:amqp, "~> 0.2.0-pre.1"},
-      {:meck, "~> 0.8.2", only: :test},
-      {:logger_file_backend, "~> 0.0.9", only: :test},
+      {:credo, "~> 0.6", only: [:dev, :test]},
+      {:dialyxir, "~> 0.4", only: [:dev], runtime: false},
       {:ex_doc, "~> 0.14", only: :dev},
+      {:excoveralls, "~> 0.5", only: :test},
+      {:logger_file_backend, "~> 0.0.9", only: :test},
+      {:meck, "~> 0.8.2", only: :test},
+      {:poison, "~> 2.0"},
```

Comment:
Could you change the order of dependencies with the following rule?

1. dependencies for all environment
2. dependencies for production
3. dependencies for test
4. dependencies for dev